### PR TITLE
Refactor sample names

### PR DESCRIPTION
**Pull request by pcnfernando**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by LakshanWeerasinghe — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+#### Data Extraction (E)
+- [Extract Data from Database](extract-data-from-database): Sample code for extracting customer orders from database.
+- [Extract Information From Unstructured Data](extract-information-from-unstructured-data): Sample code for extracting required information from unstructured data using AI.
+- [Extract Data from GitHub](extract-data-from-github): Sample code samples for extracting data from GitHub.
+
+#### Data Transformation (T)
+- [Remove Duplicates](remove-duplicates): Sample code for deduplication.
+- [Remove Approximate Duplicates](remove-approximate-duplicates): Sample code for approximate deduplication.
+- [Error Correction](error-correction): Sample code for error correction in data.
+- [Clean and Standardize Data](clean-and-standardize-data): Sample code for cleaning and standardizind data.
```
Comment (pcnfernando) [inline README.md]: @LakshanWeerasinghe should be standardizing

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+- [Data Mapper](data-mapper): Sample code for data mapping between different structures.
+- [Enrich Customer Data](enrich-customer-data): Sample code to enrich customer data.
```
Comment (pcnfernando) [inline README.md]: Can you think of some other generic name may be. Since this specific case only contain concrete info on the sample. @LakshanWeerasinghe

Comment (LakshanWeerasinghe) [reply README.md]: For which sample?

Comment (pcnfernando) [reply README.md]: Enrivh Customer Data

Comment (LakshanWeerasinghe) [reply README.md]: `Enriching Data`?

Comment (pcnfernando) [reply README.md]: Enrich data

Comment (LakshanWeerasinghe) [reply README.md]: updated.

File: README.md (hunk 1 above)
Comment (LakshanWeerasinghe) [reply README.md]: updated.

**Review by LakshanWeerasinghe — APPROVED** [review]

**Pull request merged**